### PR TITLE
Improved glossary

### PR DESCRIPTION
**Pull request by Pratichhya**
In line with the baseline document available in https://openeo.org/documentation/1.0/glossary.html#eo-data-collections 
except for the data processing modes.

### CODE REVIEW

**Comment by Pratichhya** [comment]
@soxofaan , it has been quite a while for this PR. I assume the glossary was updated as you suggested. If its good to go can I merge it since it didnt have much changed file?

File: APIs/openEO/Glossary.qmd [hunk 1]
```diff
@@ -16,28 +16,34 @@ The following glossary provides an introduction to the key technical terms commo
...
+In our domain, different terms are used to describe EO data(sets). Within openEO, a **granule** (sometimes also called *item* or *asset* in the specification) typically refers to a limited area and a single overpass leading to a very short observation period (seconds) or a temporal aggregation of such data (e.g. for 16-day MODIS composites). A **collection** is a sequence of granules sharing the same product specification. It typically corresponds to the series of products derived from data acquired by a sensor on board a satellite and having the same mode of operation.
```
Comment (soxofaan) [inline APIs/openEO/Glossary.qmd:39]: > In our domain,

In the Earth Observation domain, 

> on board a satellite

on board of a satellite

File: APIs/openEO/Glossary.qmd [hunk 2]
```diff
@@ -47,10 +53,29 @@ In the EO domain, it is common to be implicit about the temporal dimension and j
 Special cases are raster and [vector datacubes](https://r-spatial.org/r/2022/09/12/vdc.html){target="_blank"}.
 Learn more about datacubes in the [datacube documentation](https://openeo.org/documentation/1.0/datacubes.html){target="_blank"}.
 
+## Vector data
+
+In general, **vector data** represent specific things (also called "features") in a space, e.g. on the surface of the Earth.
```
Comment (soxofaan) [inline APIs/openEO/Glossary.qmd:58]: This description also matches raster data

File: APIs/openEO/Glossary.qmd [hunk 3]
```diff
@@ -47,10 +53,29 @@ In the EO domain, it is common to be implicit about the temporal dimension and j
...
+In rare cases features may not have a geometry, which is often referred to as an "empty geometry".
+
+**Geometries** consist of one or more coordinates that may be connected and then form a specific type of geometry, e.g. two points can be connected to a straight line and four straight lines can be connected to rectangle.
+
+Commonly used types of geometries are:
+- Point
+- LineString (connected straight line pieces)
+- Polygon (connected straight line pieces forming a closed ring, possibly with holes - for example a triangle or rectangle)
+
+Multiple geometries of the same type can be combined into a group of geometries, e.g. a so-called "multi point" or  "multi polygon".
```
Comment (soxofaan) [inline APIs/openEO/Glossary.qmd:72]: It should be made more clear that a multi-polygon is to be considered to be a single geometry, now it looks like it is a group of geometries, like a feature collection in GeoJSON-speak

**Review by soxofaan — COMMENTED** [review]
some quick notes

File: APIs/openEO/Glossary.qmd [hunk 4]
```diff
@@ -16,28 +16,34 @@ The following glossary provides an introduction to the key technical terms commo
   a (standardized) communication protocol, for example between a client application and back-end service
 - **client**: Software tool, framework or environment that an end-user directly interacts with.
   For example: a Jupyter notebook, a Python script/application, an RStudio session, a JavaScript based web app, etc.
-- **back-end**: computer (cloud) infrastructure that takes care of the actual EO (big) data storage and processing.
-  The end-user only interacts with the back-end through a standardized API, while the operational complexity is hidden.
+- **(cloud) back-end**: server; computer infrastructure (one or more physical computers or virtual machines) used for storing EO data and processing it
+- **big Earth observation cloud back-end**: server infrastructure where industry and researchers analyse large amounts of EO data
```
[the remark this answers is not in the document]
Comment (Pratichhya) [reply APIs/openEO/Glossary.qmd:20]: Updated the sentence as suggested

File: APIs/openEO/Glossary.qmd (hunk 1 above)
Comment (Pratichhya) [reply APIs/openEO/Glossary.qmd]: corrected

File: APIs/openEO/Glossary.qmd (hunk 2 above)
Comment (Pratichhya) [reply APIs/openEO/Glossary.qmd]: updated the sentence

File: APIs/openEO/Glossary.qmd (hunk 3 above)
Comment (Pratichhya) [reply APIs/openEO/Glossary.qmd]: Rephrase it

**Comment by Pratichhya** [comment]
@soxofaan, do you have more suggestions for this PR :) ?

File: APIs/openEO/Glossary.qmd (hunk 4 above)
[the remark this answers is not in the document]
Comment (soxofaan) [reply APIs/openEO/Glossary.qmd:20]: I'm not seeing any change, have you pushed your latest changes?

File: APIs/openEO/Glossary.qmd [hunk 5]
```diff
@@ -47,10 +52,29 @@ In the EO domain, it is common to be implicit about the temporal dimension and j
...
+In rare cases features may not have a geometry, which is often referred to as an "empty geometry".
+
+**Geometries** consist of one or more coordinates that may be connected and then form a specific type of geometry, e.g. two points can be connected to a straight line and four straight lines can be connected to rectangle.
+
+Commonly used types of geometries are:
+- Point
+- LineString (connected straight line pieces)
+- Polygon (connected straight line pieces forming a closed ring, possibly with holes - for example a triangle or rectangle)
+
+A collection of geometries of the same type can be grouped together, forming what is known as a "multi-point" or "multi-polygon." It's important to understand that a multipolygon is essentially just a gathering of several polygons.
```
Comment (soxofaan) [inline APIs/openEO/Glossary.qmd:71]:
```suggestion
Further more, these base geometries can be grouped to form, for example, so-called "multi-point" or "multi-polygon" geometries. It's important to note that these multi-geometries act as single entities with regard to their associated properties.
```

Comment (Pratichhya) [reply APIs/openEO/Glossary.qmd]: updated

File: APIs/openEO/Glossary.qmd (hunk 4 above)
[the remark this answers is not in the document]
Comment (Pratichhya) [reply APIs/openEO/Glossary.qmd]: My apologies; though staged, I committed only old changes. Now, the sentence is changed and kept a single backend definition without making a distinction.

**Review by soxofaan — APPROVED** [review]

**Pull request merged**